dev-docs/publisher-api-reference.md

```diff
@@ -938,4 +939,20 @@ The example below shows how to use these methods:
 
 {% endhighlight %}
 
+<a name="module_pbjs.aliasBidder"></a>
+
+### pbjs.aliasBidder(adapterName, aliasedName)
+
+To define an alias for a bidder adapter, call this method at runtime:
+
+{% highlight js %}
+
+pbjs.aliasBidder('appnexusAst', 'newAlias');
+
+{% endhighlight %}
+
+Defining an alias can help avoid user confusion since it's possible to load an adapter with the same string name twice in different contexts (e.g., `"appnexusAst"` may be loaded once by an SSP and once by the publisher).
```

Comment:
Defining an alias can help avoid user confusion since it's possible to send parameters to the same adapter but in different contexts (e.g  The publisher uses "appnexusAst" for demand and also uses "newAlias" which is an SSP partner that uses the "appnexusAst" adapter to serve their unique demand)